packages/beeq/src/components/tab-group/bq-tab-group.tsx

```diff
@@ -209,11 +223,26 @@ export class BqTabGroup {
 
   render() {
     return (
-      <div class={{ 'bq-tab-group flex w-full': true, 'no-divider': this.disableDivider }} part="base">
-        <div class="bq-tab-group--container flex overflow-x-auto" role="tablist" part="tabs">
-          <slot />
+      <Host class={{ 'inline-block': this.orientation === 'vertical-right' }}>
+        <div
+          class={{
+            [`bq-tab-group bq-tab-group--${this.orientation} flex w-full`]: true,
```

Comment:
I think that you missed changing w-full to is-full for consistency with logical properties.